.github/workflows/tests.yml

```diff
@@ -1,19 +1,20 @@
 name: Tests
-# Tests runs different tests (test_abci_apps, test_abci_cli, test_apps)
-# This workflow runs on every push to master or release branch and  every pull requests
-# All jobs will pass without running if no *{.go, .mod, .sum} files have been modified
 on:
   pull_request:
   push:
+    paths:
```

Comment:
if this is a required job it needs to run, this is why we went with the alternative route. Not sure if its changed recently?

Reply:
oh sorry, ignore me, this is on push